llvm/test/Transforms/InstSimplify/call.ll

```diff
@@ -826,81 +826,43 @@ define <2 x i8> @rotr_zero_shift_guard_splat(<2 x i8> %x, <2 x i8> %sh) {
   ret <2 x i8> %s
 }
 
-; If first two operands of funnel shift are undef, the result is undef
```

Comment:
Shouldn't we still keep the test coverage?

Reply:
Poison value handling has been tested by the following tests `fshl_ops_poisonXXX`. Should we keep these undef tests even if we finally remove `undef`?